.github/workflows/current.yml

```diff
@@ -0,0 +1,53 @@
+name: Test on current Node.js
```

Comment:
Could we put this job in the existing `test.yml` file?  If not, could we pick a more meaningful name for this file?  (e.g. `test_against_current_node` or `scheduled_test`)

Reply:
I believe to have a different [trigger](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#on), the jobs must be in a different file. It'd definitely be possible to run it on PRs from the existing file!

I'll rename it.